boxoffice/templates/ticket_reassignment_mail.html.jinja2

```diff
@@ -87,8 +87,8 @@
           </td>
 
           <td class="details-wrapper">
-            <p>Your ticket has been transfered to {{ new_assignee.fullname }}, by {{ current_auth.user.fullname }}.</p>
-            <p>If you believe this was because of an error, please reach out to us at <a href="mail:support@hasgeek.com">support@hasgeek.com</a>.</p>
+            <p>Your ticket has been transfered to {{ new_assignee.fullname }}{% if current_auth.user %}, by {{ current_auth.user.fullname }}{% endif %}.</p>
```

Comment:
This needs `{% trans %}` tags (in future if not now). That means the text has to be a single sentence and the `if` statement has to be outside the trans tag.